src/clr-angular/utils/drag-and-drop/interfaces/draggable-event.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export interface ClrDraggableEvent {
+    // TODO: this interface will be expanded and customized as the implementation progresses.
+    dragDataStore: any;
```

Comment:
What does `dragDataStore` do?

Reply:
This will carry the payload from draggable to droppable. https://html.spec.whatwg.org/multipage/dnd.html#the-drag-data-store

Reply:
would be beneficial if you add this as a comment in the code.